### DISCUSSION TITLE
v2.0.29

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#317] [Announcements]
## Changes

- Revert back to old NimBLE and use what works for WLED @DTTerastar (#309)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.0.29'>v2.0.29</a>.</em>

**Comment by DTTerastar** [comment]
I did find what I believe to be the root cause of the regressions when I enabled the Query feature on 2.0.  Basically I moved build_flags to a common section, but didn't do it correctly and it ended up turning off a needed build flag which led to threading issues between cores.  I was pulling my hair out trying things when they weren't actually taking effect which is why it took so long to fix. Sorry for the issues.  Obviously once we get a stable version we will be needing to make a "prerelease" channel for the adventurous so we all don't repeat this again...

**Reply by mk-maddin** [reply]
Thanks for the update - I am still seeing no devices being reported.
Is there anything that needs to be done after update of 2.0.29 except a restart to make devices working again?
Resetting exclude or something?

**Comment by xwox** [comment]
Autoupdate to 2.0.29 on 7 ESP32 devices:
No, it still does not work. Only Apple devices are recognized.
MiBand and Samsung watch are not visible in MQTT.
Everything worked well prior to 2.0.26
